satpy/demo/__init__.py

```diff
@@ -54,9 +54,10 @@
 import logging
 
 LOG = logging.getLogger(__name__)
+SATPY_DEMO_DATA_DIR = os.environ.get('SATPY_DEMO_DATA_DIR', '.')
```

Comment:
How about using `satpy.config.get('demo_data_dir')` in the functions that need this (`if base_dir is None: base_dir = satpy.config.get('demo_data_dir')`). With the way donfig works you wouldn't even have to change the environment variable name.

Reply:
sure